Detectors/TPC/simulation/run/convertDigitsToRawZS.cxx

```diff
@@ -106,13 +96,37 @@ void convertDigitsToZSfinal(std::string_view digitsFile, std::string_view output
       writer.registerLink(feeid, cruID, defaultLink, j & 1, fmt::format("{}cru{}.raw", outDir, cruID));
     }
   }
-  for (Long64_t ievent = 0; ievent < treeSim->GetEntries(); ++ievent) {
-    treeSim->GetEntry(ievent);
 
```

Comment:
regardless on using sectorBySector or not, you can disable all unneeded branches as
````
treeSim->SetBranchStatus("*",0);
treeSim->SetBranchStatus("TPCDigit_*",1);
````